app/experimenter/settings.py

```diff
@@ -428,6 +428,21 @@
 GS_PROJECT_ID = "experiments-analysis"
 GS_BUCKET_NAME = "mozanalysis"
 
+# User uploads, e.g. branch screenshots
+UPLOADS_FILE_STORAGE = config(
+    "UPLOADS_FILE_STORAGE",
+    default=DEBUG
+    and "django.core.files.storage.FileSystemStorage"
+    or "storages.backends.gcloud.GoogleCloudStorage",
+)
+UPLOADS_FILE_ROOT = config("UPLOADS_FILE_ROOT", default=MEDIA_ROOT)
+UPLOADS_FILE_BASE_URL = config("UPLOADS_FILE_BASE_URL", default=MEDIA_URL)
+UPLOADS_GS_PROJECT_ID = config("UPLOADS_GS_PROJECT_ID", "nimbus-experimenter-media-dev")
+UPLOADS_GS_BUCKET_NAME = config(
+    "UPLOADS_GS_BUCKET_NAME", "nimbus-experimenter-media-dev-uploads"
+)
+UPLOADS_GS_CREDENTIALS = config("UPLOADS_GS_CREDENTIALS", default=None)
```

Comment:
Not sure how we configured the credentials for the other GCP bucket, so this might need tweaking when we get closer to real deployment. Locally, I just crammed the whole credentials JSON blob without newlines into `.env`

Reply:
Oh right yeah I saw that.  For the results data they did this:

https://github.com/mozilla/experimenter#google-credentials

and then you can look up in the code where it's being loaded, they must have had ops set up the container with a sidecar to provide that file.  We might be able to do the same thing here, maybe ask @emtwo ?

Reply:
Switched over to using a file path here, works locally and hopefully works in deployment